tests/test_eval_detection.py

```diff
@@ -0,0 +1,95 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection evaluation unit tests"""
+
+import logging
+import pathlib
+
+import numpy as np
+import pytest
+from pandas.core.frame import DataFrame
+from scipy.spatial.transform import Rotation as R
+
+from argoverse.data_loading.object_label_record import ObjectLabelRecord
+from argoverse.evaluation.detection_utils import DistFnType, SimFnType, dist_fn, sim_fn
+from argoverse.evaluation.eval_detection import DetectionCfg, DetectionEvaluator
+
+TEST_DATA_LOC = pathlib.Path(__file__).parent.parent / "tests" / "test_data" / "detection"
+logging.getLogger("matplotlib.font_manager").disabled = True
+
+
+@pytest.fixture
+def evaluator() -> DetectionEvaluator:
+    detection_cfg = DetectionCfg(significant_digits=32)
+    return DetectionEvaluator(
+        TEST_DATA_LOC / "detections", TEST_DATA_LOC, TEST_DATA_LOC / "test_figures", detection_cfg
+    )
+
+
+@pytest.fixture
+def metrics(evaluator: DetectionEvaluator) -> DataFrame:
+    return evaluator.evaluate()
+
+
+def test_center_similarity() -> None:
+    olr1 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([0, 0, 0]), 5.0, 5.0, 5.0, 0)])
+    olr2 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([3, 4, 0]), 5.0, 5.0, 5.0, 0)])
+    assert sim_fn(olr1, olr2, SimFnType.CENTER) == -5
+
+
+def test_iou_2d_similarity() -> None:
+    # TO DO
+    pass
+
+
+def test_iou_3d_similarity() -> None:
+    # TO DO
+    pass
+
+
+def test_translation_distance() -> None:
+    df1 = DataFrame([{"translation": [0.0, 0.0, 0.0]}])
+    df2 = DataFrame([{"translation": [5.0, 5.0, 5.0]}])
+    assert dist_fn(df1, df2, DistFnType.TRANSLATION) == 75 ** (1 / 2)
+
+
+def test_scale_distance() -> None:
```

Comment:
1-line docstring would be helpful to explain the scenario and goal of the test

Reply:
What sort of documentation were you thinking of? I think comments inline in the code are great, and then in the EvalAI Evaluation tab we should add the equivalent of https://evalai.cloudcv.org/web/challenges/challenge-page/453/evaluation.

Reply:
Let's add a README in the evaluation folder.